### ISSUE TITLE
RabbitMQ native support

### ISSUE THREAD

**Issue by ppalaga** [apache/camel-quarkus#1531]
Contributions are welcome. Adding an integration test would the the first step. ActiveMQ is a good place to look how such an integration test should look like https://github.com/apache/camel-quarkus/tree/master/integration-tests/activemq

**Comment by Jeansen** [comment]
@ppalaga That was a quick reply ;-) I've already checked how the project is setup with extensions for quarkus and JVM. I also thought that the ActiveMQ extension would be a candidate to start with. Currently I am all new to Quarkus and Camel. But maybe I'll be able to contribute something.

So far I've already created a simple project with a camel route on my local system. Building native worked and the program spits out the same text when run in JVM (dev) mode.

**Comment by ppalaga** [comment]
Feel free to ask if you have specific questions. We also have a step by step guide for promoting extensions from JVM to native https://camel.apache.org/camel-quarkus/latest/contributor-guide/promote-jvm-to-native.html

**Comment by Jeansen** [comment]
Oh, great. I'll check it out!

**Comment by ppalaga** [comment]
> here is perhaps some potential to state the relationship between `camel.component.*` props and `camel-quarkus-main` even clearer. We should perhaps also mention it somewhere in the contributor guide.

Addressed in https://github.com/apache/camel-quarkus/pull/1579/commits/9752ff694fd4ad5607be25ac69c5481789930fe7

**Comment by ppalaga** [comment]
Resolved via #1567

**Comment by ppalaga** [comment]
Thanks again, @Jeansen !

**Comment by Jeansen** [comment]
Shouldn't it be version 1.1.0? 
https://camel.apache.org/camel-quarkus/latest/reference/extensions/rabbitmq.html
https://github.com/apache/camel-quarkus/milestone/5?closed=1

**Comment by ppalaga** [comment]
Well, native support is indeed since 1.1.0, but the JVM-only extension was there since 1.0.0. So the extension as a whole is since 1.0.0. Should we make this more explicit on the web page?

**Comment by Jeansen** [comment]
Hm, I am not sure. It might be of interest. At the moment there are not that many versions but in the future it might be of interest since which version native support is available. There might be situations where users simply cannot use the latest versions but have to stick to a specific version. So the argument would be not to use the latest but at least a the version that also supports native. 

Long story short: I would say yes and suggest to also ad a column for native :-)

**Comment by ppalaga** [comment]
I think you are making a valid point. Could you please file a new issue for that?
